exercises/E-conditionals/exercise.js

```diff
@@ -6,8 +6,18 @@
   If Daniel is a student, print out "Hi, I'm Daniel, I'm a student."
 */
 
-let name = "Daniel";
-let danielsRole = "mentor";
+function Role(name, danielsRole) {
+ 
+  if (name == "Daniel" && danielsRole == "mentor"){
+    return "Hi, I'm Daniel, I'm a mentor.";
+  }if (danielsRole == "student"){
```

Comment:
Please place the next 'if' on the following line. It would make the code read easier.